backend/test/controllers/user.test.js

```diff
@@ -0,0 +1,53 @@
+const { getAll, getUser, addUser, updateUser, deleteUser } = require("../../src/controllers/user");
+
+describe("Testing user controllers", () => {
+    it("Gets an array of all users", async () => {
+        const result = await getAll();
+
+        expect(Array.isArray(result)).toBe(true);
+        result.forEach((userObject) => expect("id" in userObject).toBe(true));
+    });
+
+    it("Gets an individual user by ID.", async () => {
```

Comment:
I think you need to mock the response from the request, otherwise what is the point, eventually this won't be hardcoded, and will hold little value.